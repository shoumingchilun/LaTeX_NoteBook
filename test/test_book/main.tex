%-*- coding: UTF-8 -*-
% gougu.tex
% 勾股定理
\documentclass[UTF8]{ctexart}
\usepackage{graphicx, float, hyperref}

\title{杂谈勾股定理}
\author{张三}
\date{\today}

\bibliographystyle{plain}

\begin{document}

\maketitle
\tableofcontents
\section{勾股定理在古代}
西方称勾股定理为毕达哥拉斯定理，将勾股定理的发现归功于公元前␣6␣世纪的毕达哥拉斯学派。该学派得到了一个法则，可以求出可排成直角三角形三边的三元数组。毕达哥拉斯学派没有书面著作，\cite{Kline}
\begin{figure}[h]
    \centering
    \includegraphics[width=3cm]{不要打脸.jpg}
    \caption{滚}
\end{figure}
该定理的严格表述和证明则见于欧几里德《几何原本》的命题\ 47：“直角三角形斜边上的正方形等于两直角边上的两个正方形之和。”证明是用面积做的。
我国《周髀算经》载商高（约公元前\ 12\ 世纪）答周公问…… \cite{quanjing}

\bibliography{math}
\end{document}
