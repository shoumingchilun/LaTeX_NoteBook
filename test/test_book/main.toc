\contentsline {section}{\numberline {1}勾股定理在古代}{1}{section.1}%
