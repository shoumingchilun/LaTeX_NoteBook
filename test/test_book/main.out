\BOOKMARK [1][-]{section.1}{\376\377\122\376\200\241\133\232\164\006\127\050\123\344\116\343}{}% 1
